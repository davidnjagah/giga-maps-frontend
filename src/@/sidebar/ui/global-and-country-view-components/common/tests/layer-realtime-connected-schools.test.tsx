import { render } from '@testing-library/react';
import { testWrapper } from '~/tests/jest-wrapper';
import LayerRealtimeConnectedSchools from '../layer-realtime-connected-schools.view';

describe('Layer realtime connected school', () => {
  it('should render component', () => {
    const isLoading = true;
    const { asFragment } = render(testWrapper(<LayerRealtimeConnectedSchools isLoading={isLoading} />));
    expect(asFragment).toMatchSnapshot();
  });

  it('should render component', () => {
    const isLoading = false;
    const { asFragment } = render(testWrapper(<LayerRealtimeConnectedSchools isLoading={isLoading} />));
    expect(asFragment).toMatchSnapshot();
  });
});
