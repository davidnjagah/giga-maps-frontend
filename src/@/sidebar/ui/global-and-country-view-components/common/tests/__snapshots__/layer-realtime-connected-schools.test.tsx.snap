// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layer realtime connected school should render component 1`] = `[Function]`;

exports[`Layer realtime connected school should render component 2`] = `[Function]`;
